moped-editor/src/components/GridTable/Filters.js

```diff
@@ -122,24 +121,21 @@ const Filters = ({
    * @default
    */
   const classes = useStyles();
-  let [, setSearchParams] = useSearchParams();
+  let [searchParams, setSearchParams] = useSearchParams();
 
   const { loading, error, data } = useQuery(LOOKUP_TABLES_QUERY);
 
   if (error) console.error(error);
 
-  /* Consume existing filters or start with an empty filter if none exist */
-  const initialFilterParameters = useMemo(() => {
-    if (filters.length > 0) {
-      return filters;
-    } else {
-      return [generateEmptyFilter()];
-    }
-  }, [filters]);
+  /* Consume existing URL search params or start with an empty filter if none exist */
+  const initialFilterParameters = useMakeFilterState(
+    searchParams,
+    advancedSearchFilterParamName
+  );
```

Comment:
this keeps advanced search filter changes from persisting when the user exits out without applying. now when this component renders it is pulling from the URL search params initially